File: job-tracker-frontend/src/types/job.ts
export interface Job {
    id: number;
    company: string;
    title: string;
    status: string;
    appliedDate: string;
    notes: string;
}
export interface JobFormData {
    company: string;
    title: string;
    status: string;
    appliedDate: string;
    notes: string;
}

File: JobTrackerAPI/Models/Job.cs
namespace JobTrackerAPI.Models;

public class Job
{
    public int Id { get; set; }
    public string Company { get; set; } = "";
    public string Title { get; set; } = "";
    public string Status { get; set; } = "Applied";
    public DateTime AppliedDate { get; set; }
    public string Notes { get; set; } = "";
}
